app/assets/javascripts/lesson_planner/lesson_planner.jsx

```diff
@@ -188,6 +188,19 @@ EC.LessonPlanner = React.createClass({
     });
   },
 
+	// TODO: remove staging from the URL or build a regex that grabs everythig before the third slash using window.location.href
+	getInviteStudentsUrl: function() {
+		return ('/teachers/classrooms/' + $(".tab-pane").data().classroomId + '/invite_students');
+	},
+
+	studentsPresent: function() {
+		return $(".tab-pane").data().students;
+	},
+
+	getLastClassroomName: function() {
+		return $(".tab-pane").data().classroomName;
```

Comment:
@RyanNovas we might want to look into pulling this data using ajax.
Generally I think it might be best to reserve data handed from views to react to be only that data which is needed in order to make the requisite ajax calls (such as the id of the teacher). Id be down to look into what this would look like in this particular case.
Also I think its best - when data is attached to html elements in the view - to use more revealing classnames like '#lesson-planner' instead of something generic like '.tab-pane',